src/Components/Controls/Controls.module.css

```diff
@@ -0,0 +1,19 @@
+.control-title {
```

Comment:
I don't think this naming of classes works with modules? Can you confirm this works?

Reply:
Yes it's just ugly